lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -55,6 +55,15 @@ function dosomething_reportback_build_reportbacks_query($params = array()) {
     }
   }
 
+  if (!isset($params['flagged'])) {
```

Comment:
This block looks like it could be simplified to
```
if (!isset($params['flagged']) || !params['flagged']) {
  $query->condition('rb.flagged', 0);
}
```